### PR TITLE
Adds new character sheet - Alternate version

### LINKED ISSUE #445: Adds new character sheet - Alternate version
**Motivation / Use Case**

Add new character sheet - Alternate Version

### PR DESCRIPTION
**Pull request by Jessomadic**
## Description: Adds new character sheet
Alternate version

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] I have commented my code, particularly in hard-to-understand areas
  - [x] I have made corresponding changes to the documentation if necessary
  - [x] There is no commented out code in this PR.
  - [x] My changes generate no new warnings (check the console)

### CODE REVIEW

**Comment by datdamnzotz** [comment]
fillable-char-sheetstyle-2 are not populating spells.
image
Missing
image
Missing
image

**Comment by datdamnzotz** [comment]
Features and traits in fillable-char-sheetstyle-3 spells are not populating.
image

Same with current hitpoints
image

**Comment by Jessomadic** [comment]
That's strange. Is that happening on all spell sheet numbers or just one? For the style 3 features and the style 2 spells. I have to head into work for a few hours but I'll be sure to look over everything again once I am done

**Comment by datdamnzotz** [comment]
> That's strange. Is that happening on all spell sheet numbers or just one? For the style 3 features and the style 2 spells. I have to head into work for a few hours but I'll be sure to look over everything again once I am done

That was a level 20 wizard - so its pulling the 6th sheet. But need to check them all to make sure the fields are correct.

**Comment by Jessomadic** [comment]
> > That's strange. Is that happening on all spell sheet numbers or just one? For the style 3 features and the style 2 spells. I have to head into work for a few hours but I'll be sure to look over everything again once I am done
> 
> 
> 
> That was a level 20 wizard - so its pulling the 6th sheet. But need to check them all to make sure the fields are correct.

I will definitely take a look. I made one that was a level 20 wizard and a level 20 warlock to make sure everything worked and it did but I might have missed something.

**Comment by Jessomadic** [comment]
image The Hitpoint field is 100% correct on the 3rd Style sheet. It might be something on your end? @datdamnzotz

**Comment by Jessomadic** [comment]
Likewise the Spellsheets on style 2 is correct. However more fields broke for that second page and will be fixed
image

File: src/cljc/orcpub/pdf_spec.cljc [hunk 1]
```diff
@@ -546,7 +546,8 @@
       :hd total-hit-dice
       :initiative (common/bonus-str (es/entity-val built-char :initiative))
       :speed speed
-      :hp-max (es/entity-val built-char :max-hit-points)
+      :hp-max (es/entity-val built-char :-max-hitpoints)
```
Comment (datdamnzotz) [inline src/cljc/orcpub/pdf_spec.cljc]: Oops shouldn't rename this!!

**Review by datdamnzotz — APPROVED** [review]
👍 smoke tests look good - release to the wild

**Pull request merged**